### PR TITLE
KHADRI-220924-102-PM UPDATE AND DELETE OPERATIONS APPEND ON MOBILE MODULE

### PR DESCRIPTION
**Pull request by thasmiyas39**

### CODE REVIEW

**Comment by thasmiyas39** [comment]
resolved conflicts and modified Mobile Module

File: SQL/scripts.sql [hunk 1]
```diff
@@ -1,7 +1,7 @@
 create Schema 2024_batch_jdbc;
 use 2024_batch_jdbc;
 create table Employee (id int,name varchar(20),desigination varchar(30),salary double);
-create table customer(id int,name varchar(70), address varchar(100), phone_Num long);
```
Comment (khadri2021) [inline SQL/scripts.sql:4]: why this change?

File: db.properties [hunk 2]
```diff
@@ -1,3 +1,3 @@
 URL = jdbc:MYSQL://localhost:3306/2024_batch_jdbc
 USER = root
-PASSWORD = Wellcome@123
```
Comment (khadri2021) [inline db.properties:3]: why this change?

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java [hunk 3]
```diff
@@ -22,6 +22,9 @@ public class JdbcDataDriver {
 	private CustomerDao custDao;
 	private MobileDao mobileDao;
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java:23]: why you need mobileDao in Driver class?

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java [hunk 4]
```diff
@@ -103,17 +100,23 @@ private void process() {
...
 				int custRowCount = 1;
 				for (int i = 0; i < custCount; i++) {
-					custProcessor.process(custRowCount);
+					custProcessor.insertProcess(custRowCount);
 					custRowCount++;
 				}
 			} else if (operationTypeCustomer == 2) {
 				System.out.println("fetching customer records from the database...");
-				custDao.customerSelectData();
+				custDao.selectCustomerData();
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java:122]: use processor to call selectCustomerData not dao

File: SQL/scripts.sql (hunk 1 above)
Comment (thasmiyas39) [reply SQL/scripts.sql:4]: i didnot change anything here

File: db.properties (hunk 2 above)
Comment (thasmiyas39) [reply db.properties:3]: changed root as password

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java (hunk 4 above)
Comment (thasmiyas39) [reply src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java:122]: it is not my module to change

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java (hunk 3 above)
Comment (thasmiyas39) [reply src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java:23]: it is a variable for internally creating object

Comment (thasmiyas39) [reply src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java:23]: > it is a variable for internally creating an object

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java (hunk 4 above)
Comment (thasmiyas39) [reply src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java:122]: this is not my module to change
